partcad/src/partcad/adhoc/convert.py

```
@@ -0,0 +1,86 @@
+from pathlib import Path
+import shutil
+import tempfile
+from partcad.context import Context
+
+
+def convert_cad_file(input_filename, input_type, output_filename, output_type):
```

Ruff (0.8.2)

Missing return type annotation for public function `convert_cad_file`

Add return type annotation: `None`

(ANN201)